### PR TITLE
Stop project leaks in tests by removing call to myFixture.addFileToProject

### LINKED ISSUE #1310: Fix flakiness in tests inheriting from RestSignatureInspectuionTest
This only affects 2017 branches, and is blocking our release of a 2017+ version of our plugin.

### PR DESCRIPTION
**Pull request by joaoandremartins**
Fixes #1310

### CODE REVIEW

File: google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java [hunk 1]
```diff
@@ -51,10 +51,6 @@ protected void tearDown() throws Exception {
     }
     finally {
       super.tearDown();
-
-      if (myFixture != null && getProject() != null && !getProject().isDisposed()) {
-        Disposer.dispose(getProject());
```
Comment (etanshaul) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java]: Isn't there an import you can clean up too?

**Comment by etanshaul** [comment]
Just for documentation purporses:

We discovered by examining strong paths to GC roots that it was the PsiFile causing the leak - ultimately causing Project objects to be held around. Furthermore, it turns out that these 2 lines setting up the PsiFile are not need for the unit test (perhaps they were left around after a refactor).

File: google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java (hunk 1 above)
Comment (joaoandremartins) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java]: Nice! You're getting better at detecting unused imports than the IJ analysis tool :D

Comment (etanshaul) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java]: lol

Comment (etanshaul) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java]: on that note, we once were failing builds on unused imports. worth looking into why this isn't happening anymore.

**Pull request merged**